Python/compile.c

```diff
@@ -1609,12 +1694,18 @@ cfg_builder_addop_j(cfg_builder *g, location loc,
 {
     assert(IS_LABEL(target));
     assert(IS_JUMP_OPCODE(opcode) || IS_BLOCK_PUSH_OPCODE(opcode));
-    return cfg_builder_addop(g, opcode, target.id, loc);
+    return cfg_builder_addop(g, opcode, target.id, loc,
+                             UNUSED_OPARG, UNUSED_OPARG, UNUSED_OPARG);
 }
 
 #define ADDOP(C, LOC, OP) \
     RETURN_IF_ERROR(cfg_builder_addop_noarg(CFG_BUILDER(C), (OP), (LOC)))
 
+#define ADDOP_REGS(C, LOC, OP, R1, R2, R3) { \
+    if (!cfg_builder_addop(CFG_BUILDER(C), (OP), 0, (LOC), (R1), (R2), (R3))) \
```

Comment:
The check for error is wrong here, you need to check if it is < 0.
```suggestion
    if (cfg_builder_addop(CFG_BUILDER(C), (OP), 0, (LOC), (R1), (R2), (R3)) < 0) \
```

Reply:
Ah! Thanks, that's a merge conflict.